src/js/index.js

```diff
@@ -64,9 +72,12 @@ function handleTags() {
 	if (!search) {
```

Comment:
I added it since the website might be used from a browser that doesn't support `URLSearchParams`, but I guess that in that case the website wouldn't work anyways.